rpmio/rpmfileutil.cc

```diff
@@ -306,79 +307,34 @@ char *rpmCleanPath(char * path)
     return path;
 }
 
-/* Merge 3 args into path, any or all of which may be a url. */
+/* Merge 3 args into path */
```

Comment:
Did we drop support for URLs?

Reply:
Not really, the URL support never worked at all (as is explained in the commit message and the API docs update). Any URL's in there and the output is utterly useless garbage.

Reply:
So it was a lucky case of being SO broken nobody could have possibly used it for URL's, so we can just prune that cruft.

Reply:
So I guess stuff like `rpm -ivh https://kojipkgs.fedoraproject.org//packages/rpm/4.20.0/1.fc42/src/rpm-4.20.0-1.fc42.src.rpm` are handled elsewhere?

Reply:
Yes.